src/Resources/views/datatable_html.html.twig

```diff
@@ -7,23 +7,24 @@
             <th>{{ column.label|trans }}</th>
         {% endfor %}
     </tr>
-    {#% if datatable.option('searching') and datatable.setting('column_filter') in ['both', 'thead'] %}
+    {% if datatable.option('searching') and columnFilter in ['both', 'thead'] %}
         <tr class="datatable-filters">
             {% for column in datatable.columns %}
                 <td>{% if column.filter != null %}{% include column.filter.templateHtml %}{% endif %}</td>
```

Comment:
if template is included in both head and foot, then widget ids will collide, we should pass a context var like 
```twig
{% include column.filter.templateHtml with {'location': 'thead'} %}
```
or use a class instead of an id

Reply:
Proper use of classes is preferable as it also makes the CSS selectors elsewhere easier, and facilitates styling better.